Sources/Rendering/OpenGL/Framebuffer/index.js

```diff
@@ -242,7 +247,8 @@ export function extend(publicAPI, model, initialValues = {}) {
   // Build VTK API
   macro.obj(publicAPI, model);
 
-  macro.setGet(publicAPI, model, ['colorTexture']);
+  model.colorTextures = [];
```

Comment:
by doing so, you prevent passing color textures in constructor.
```
function defaultValues (initialValues) {
  return {
  // openGLRenderWindow: null,
  // glFramebuffer: null,
  colorTextures: [],
  // depthTexture: null,
  previousDrawBinding: 0,
  previousReadBinding: 0,
  prev...,
  ..initialValues
  };
}
function extend(...) {
  Object.assign(model, defaultValues(initialValues));
```

Reply:
Updated to error on this case and cleaned up the naming and added some error checks.